configs/xcache-redir/config.d/03-redir-tunning.cfg

```diff
@@ -0,0 +1,10 @@
+# Retries should actually be 0, but 1 is minimum                                              
```

Comment:
```suggestion
#
# Tuning configuration for an XCache redirector
#
# **********************************************************************
# * WARNING: DO NOT EDIT THIS FILE.  IT WILL BE OVERWRITTEN ON UPGRADE *
# **********************************************************************
#
# Instead of editing this file,
# - Override OSG-provided defaults by dropping a site-specific configuration
#   file in `/etc/xrootd/config.d` and prefixing it with "90-" or higher.
#

# Retries should actually be 0, but 1 is minimum                                              
```